.travis.yml

```diff
@@ -30,3 +30,4 @@ script:
 
 after_success:
   - java -cp ~/codacy-coverage-reporter-assembly-latest.jar com.codacy.CodacyCoverageReporter -l Java -r build/reports/jacoco/jacocoTestReport.xml
+  - test "$TRAVIS_BRANCH" = "master" && test "$TRAVIS_PULL_REQUEST" = "false" && ./gradlew publish
```

Comment:
what's this doing and where are they keys setup?
this is how the other repos have been done:
https://github.com/hmcts/cmc-pdf-service-client/blob/master/.travis.yml